docs/demos/hello_world_demo.md

```diff
@@ -356,6 +356,10 @@ Let's look how each of these objects are implemented. If you have fetched the He
           if self.collection_threshold_reached:
               synchronized_data = self.synchronized_data.update(
                   participants=self.collection,
```

Comment:
`participants=self.collection,` is in fact `participant_to_selection=self.collection,`. The `participants` are the `keys()`.